CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Number Recycling
+
+## Table of Contents
+
+- [v0.1.0](#v010)
+
+**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
+
+# v0.1.0
+
+**This is the first alpha version of the KYC APIs.** 
+
+- API [definition](https://github.com/camaraproject/NumberRecycling/tree/main/code/API_definitions)
```

Comment:
Link should be to release tag (r1.1) rather than main branch
```suggestion
- [number-recycling v0.1.0-alpha.1](https://github.com/camaraproject/NumberRecycling/blob/r1.1/code/API_definitions/number-recycling.yaml)
```